src/server/api/root.ts

```diff
@@ -1,7 +1,7 @@
 import { exampleRouter } from "~/server/api/routers/example";
 import { createCallerFactory, createTRPCRouter } from "~/server/api/trpc";
 import { applicationRouter } from "./routers/application";
-import { loginRouter } from "./routers/login";
+import { login } from "./routers/login";
```

Comment:
can we just rename this to loginRouter in `./routers/login.ts`? Just for consistency.

Reply:
done